experimental/parser/lex_number.go

```diff
@@ -142,6 +139,8 @@ func lexRawNumber(l *lexer) token.Token {
 
 	for !l.Done() {
 		r := l.Peek()
+		//nolint:gocritic // This trips a noisy "use a switch" lint that makes
+		// this code less readable.
```

Comment:
Yeah, it's a shame that it doesn't recognize "break" in an else case, which gets to be a pain to do inside of a switch since it would have to break the switch and break again outside the switch :/

Reply:
go having break-in-switch is so bone-headed.